backend/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "remoteit-headless",
-  "version": "2.5.24-alpha.10",
+  "version": "2.5.24-alpha.11",
```

Comment:
You don't need to increment every commit. I'll increment and tag right before I do a build.